### DISCUSSION TITLE
'Index was out of range'

### DISCUSSION THREAD

**Discussion by SophiaUCL** [bonsai-rx/bonsai#2057] [Q&A]
Hi, I'm running Sleap on Bonsai and I'm running:
FileCapture >> Grayscale >> PredictPoses >> Index

I get the following error: Index was out of range. Must be non-negative and less than the size of the collection.

This is despite my index being 0. Does anyone know how to fix this?
Thank you!

**Comment by bruno-f-cruz** [comment]
Your model is likely not finding any instance in the image and the list is returned as empty. You should place a condition that drops lists with count 0 before indexing the 0th element.

**Comment by SophiaUCL** [comment]
Thank you so much! This solved the issue. I applied an Expression Condition with it.Count != 0
________________________________
